packages/thumbprint-react/components/Avatar/index.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import * as tokens from '@thumbtack/thumbprint-tokens';
```

Comment:
Does Rollup handle this type of import well? You can check by running `yarn run build:dist` in the Thumbprint root folder and then checking the contents of `packages/thumbprint-react/dist` to make sure that it didn't pull in all of the tokens as part of the bundling.

Reply:
seems like rollup is smart. the dist directly it gets compiled to this

```
import { tpColorGray200, tpColorWhite, tpColorIndigo600, tpColorGreen600, tpColorYellow600, tpColorRed600, tpColorPurple600, tpColorBlue600, tpColorIndigo100, tpColorGreen100, tpColorYellow100, tpColorRed100, tpColorPurple100, tpColorBlue100 } from '@thumbtack/thumbprint-tokens';
```

Reply:
Ah, wow. That's neat. 👍